5.5/examples/replica/mysql_replica.json

```diff
@@ -158,11 +158,11 @@
                 "env": [
                   {
                     "name": "MYSQL_MASTER_USER",
```

Comment:
@bparees after our conversation yesterday on IRC I decided to step back on changing the parameter name.